### PR TITLE
temporary changes to plot rxd in panel 3

### PR DESCRIPTION
**Pull request by salvadord**
this is just temporary so can see the rxd plot properly... don't freak out! ;)

### CODE REVIEW

File: netpyne_ui/netpyne_geppetto.py [hunk 1]
```diff
@@ -331,6 +331,16 @@ def getNetPyNELFPLocationsPlot(self):
             svgs.append(ui.getSVG(fig[0][0]))
         
         return svgs.__str__()
+
+    def getNetPyNERxDConcentration(self):
```
Comment (adrianq) [inline netpyne_ui/netpyne_geppetto.py]: @salvadord Can we name this method following the standard, i.e. 'getNetPyNERxDConcentrationPlot'?

File: netpyne_ui/netpyne_geppetto.py [hunk 2]
```diff
@@ -331,6 +331,16 @@ def getNetPyNELFPLocationsPlot(self):
...
+        try:
+            fig = rxd.plotExtracellularConcentration(species=rxd.ca)
```
Comment (adrianq) [inline netpyne_ui/netpyne_geppetto.py]: Is it not needed to import rxd for this to work?

File: netpyne_ui/netpyne_geppetto.py (hunk 1 above)
Comment (salvadord) [reply netpyne_ui/netpyne_geppetto.py]: yes, can rename to that

File: netpyne_ui/netpyne_geppetto.py (hunk 2 above)
Comment (salvadord) [reply netpyne_ui/netpyne_geppetto.py]: this is just a temporary hack for the tutorial, so for this to work the user has to run from the jupyter nb: `import gui_tut3_rxd as rxd`

Comment (adrianq) [reply netpyne_ui/netpyne_geppetto.py]: Actually, you can just do 'from neuron import crxd as rxd', can't you? and we are already doing sth similar with matplotlib here https://github.com/MetaCell/NetPyNE-UI/blob/e00052075a5ce739381998fc799c943f0b01325f/netpyne_ui/netpyne_geppetto.py#L21

If what I am saying is correct is almost the same thing we are doing with matplotlib and I wouldn't consider it as hacky solution.

Comment (salvadord) [reply netpyne_ui/netpyne_geppetto.py]: it's not the same as rxd -- I shouldn't really have called it rxd (it was just to make it shorter).
the module I'm importing is custom made for the tutorial: https://github.com/Neurosim-lab/netpyne_workspace/blob/CNS18/gui_tut3_rxd.py

**Pull request merged**